components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -78,30 +125,45 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}";
     option (chef.automate.api.iam.policy).action = "iam:policies:update";
     };
+  /*
+  Get IAM version
+
+  Returns the major and minor version of IAM that your automate installation is running.
+  */
   rpc GetPolicyVersion (GetPolicyVersionReq) returns (GetPolicyVersionResp) {
     option (google.api.http) = {
       get: "/iam/v2/policy_version"
       additional_bindings {
-        get: "/iam/v2beta/policy_version"     
+        get: "/iam/v2beta/policy_version"
       }
     };
     option (chef.automate.api.policy).resource = "auth:policies";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "iam:policyVersion";
     option (chef.automate.api.iam.policy).action = "iam:policies:get";
     };
+  /*
+  List policy members
+
+  List all members of a specific policy.
+  */
   rpc ListPolicyMembers (ListPolicyMembersReq) returns (ListPolicyMembersResp) {
```

Comment:
I'm not sure where `ListPolicyMembersResp` is defined but it is not documented in the UI.